src/server/JasmineGraphServer.cpp

```diff
@@ -3088,45 +3107,36 @@ void JasmineGraphServer::initiateFiles(std::string graphID, std::string training
 
 void JasmineGraphServer::initiateCommunication(std::string graphID, std::string trainingArgs, SQLiteDBInterface sqlite) {
 
-    Utils utils;
-    int fl_clients = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl_clients"));
-    int threadLimit = fl_clients + 1;
-    std::thread *workerThreads = new std::thread[threadLimit];
+    int count = 0;
+    JasmineGraphTrainingSchedular *schedular = new JasmineGraphTrainingSchedular();
+    map<string, map<int, int>> scheduleForAllWorkers = schedular->schedulePartitionTraining(graphID);
+    std::map<std::string, JasmineGraphServer::workerPartition> graphPartitionedHosts = this->getWorkerPartitions(
+            graphID);
+    int partition_count = graphPartitionedHosts.size();
 
-    Utils::worker workerInstance;
-    vector<Utils::worker> workerVector = utils.getWorkerList(sqlite);
+    std::cout << partition_count << std::endl;
```

Comment:
Lets remove these `std::cout` statements. If those are really required lets convert those to log statements.

Reply:
fixed

Reply:
Ok

Reply:
Ok